mindsdb/integrations/handlers/shopify_handler/README.md

```diff
@@ -168,6 +169,12 @@ AND last_name = 'Doe'
 AND email = 'john.doe@example.com';
 ~~~~
 
+~~~~sql
+DELETE FROM shopify_datasource.orders
+WHERE id=5632671580477 
+AND contact_email='egnition_sample_21@egnition.com';
```

Comment:
To delete the record you need just the id column. Please remove this line `AND contact_email='egnition_sample_21@egnition.com'` from this sample statement.